packages/react/ds/src/header/components/header-menu.tsx

```diff
@@ -16,19 +19,97 @@ type HeaderMenuProps = {
     label?: string;
     icon?: IconId;
   };
+} & Pick<HeaderProps, 'tools'>;
+
+type MenuItemAccordionProps = {
+  index: number;
+  item: { label?: string; slot: React.ReactNode };
+};
+
+const MenuListItem = ({ href = '#', label = '', bold = true }) => (
+  <a
+    aria-label={label || 'link with no label'}
+    href={href}
+    className="hover:gi-underline hover:gi-underline-offset-sm gi-block gi-py-4 gi-border-gray-100 gi-border-b-xs gi-border-solid"
+  >
+    <span className={cn('gi-text-sm', 'gi-ml-1', { 'gi-font-bold': bold })}>
```

Comment:
do we need `{ 'gi-font-bold': bold }` ? can't we do just `gi-font-bold` instead ?